app/internal/actor/attorney.go

```diff
@@ -28,49 +28,101 @@ func (a Attorney) FullName() string {
 	return fmt.Sprintf("%s %s", a.FirstNames, a.LastName)
 }
 
-type Attorneys []Attorney
+// TrustCorporation contains details about a trust corporation, provided by the applicant
+type TrustCorporation struct {
+	// Name of the company
+	Name string
+	// CompanyNumber as registered by Companies House
+	CompanyNumber string
+	// Email to contact the company
+	Email string
+	// Address of the company
+	Address place.Address
+}
+
+type Attorneys struct {
+	TrustCorporation TrustCorporation
```

Comment:
The first commit shows how it was with `TrustCorporation`/`ReplacementTrustCorporation` on the `Lpa` type. I moved it here because I wanted to get the length of attorneys, and combining them with a `Len()` method seemed the most foolproof way